clang_delta/RemoveUnusedEnumMember.cpp

```diff
@@ -79,30 +85,25 @@ void RemoveUnusedEnumMember::HandleTranslationUnit(ASTContext &Ctx)
   if (QueryInstanceOnly)
     return;
 
-  if (TransformationCounter > ValidInstanceNum) {
-    TransError = TransMaxInstanceError;
-    return;
-  }
```

Comment:
I think ```clang_delta``` and the creduce driver script have certain kind of contract based on the return status. It would be great if we could make this pass be consistent with the others, e.g. something similar to what we did for ```RemoveUnusedFunction```:

https://github.com/csmith-project/creduce/blob/master/clang_delta/RemoveUnusedFunction.cpp#L277